cmd/kk/pkg/plugins/storage/modules.go

```diff
@@ -55,6 +54,7 @@ func (d *DeployLocalVolumeModule) Init() {
 			Data: util.Data{
 				"ProvisionerLocalPVImage": images.GetImage(d.Runtime, d.KubeConf, "provisioner-localpv").ImageName(),
 				"LinuxUtilsImage":         images.GetImage(d.Runtime, d.KubeConf, "linux-utils").ImageName(),
+				"BasePath":                filepath.Join(d.KubeConf.Cluster.Storage.OpenEBS.BasePath, "local"),
```

Comment:
IMO, people maybe don't like the data dir is not the expected one caused we always add a subpath `local` here.

Reply:
Done, please review

Reply:
Thanks, it would be better if you can squash your commits.

Reply:
![image](https://user-images.githubusercontent.com/26432832/194698378-183a86c8-f772-4b0d-9549-22bfc72ff607.png)
You can select `squash and merge` when merging,and you can also [disable](https://docs.github.com/cn/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests) other options